schema/name_osm.js

```diff
@@ -38,14 +35,4 @@ var OSM_NAMING_SCHEMA = {
   // 'sorting_name':     'sorting'
 };
 
-// this property is considered the 'primary name'
```

Comment:
This was kinda weird and overly fancy, I removed it.

Reply:
Over the years we've learned many things you _can_ do with Javascript, but shouldn't :P